modules/ozoneBidAdapter.js

```diff
@@ -535,26 +601,31 @@ export const spec = {
   },
   /**
    *  Look for pubcid & all the other IDs according to http://prebid.org/dev-docs/modules/userId.html
+   *  NOTE that criteortus is deprecated & should be removed asap
    *  @return map
    */
   findAllUserIds(bidRequest) {
     var ret = {};
-    let searchKeysSingle = ['pubcid', 'tdid', 'parrableId', 'idl_env', 'digitrustid', 'criteortus'];
+    // @todo - what is fabrick called & where to look for it? If it's a simple value then it will automatically be ok
+    let searchKeysSingle = ['pubcid', 'tdid', 'id5id', 'parrableId', 'idl_env', 'criteoId', 'criteortus',
+      'sharedid', 'lotamePanoramaId', 'fabrickId'];
     if (bidRequest.hasOwnProperty('userId')) {
       for (let arrayId in searchKeysSingle) {
         let key = searchKeysSingle[arrayId];
         if (bidRequest.userId.hasOwnProperty(key)) {
-          ret[key] = bidRequest.userId[key];
+          if (typeof (bidRequest.userId[key]) == 'string') {
+            ret[key] = bidRequest.userId[key];
+          } else if (typeof (bidRequest.userId[key]) == 'object') {
+            ret[key] = bidRequest.userId[key][Object.keys(bidRequest.userId[key])[0]]; // cannot use Object.values
+          } else {
+            this.logError(`failed to get string key value for userId : ${key}`);
+          }
         }
       }
       var lipbid = utils.deepAccess(bidRequest.userId, 'lipb.lipbid');
       if (lipbid) {
         ret['lipb'] = {'lipbid': lipbid};
       }
-      var id5id = utils.deepAccess(bidRequest.userId, 'id5id.uid');
-      if (id5id) {
-        ret['id5id'] = id5id;
-      }
```

Comment:
this should not have been removed; or the code above needs to handle the fact that the id5id is an object not a string